rtbkit/core/router/filter_pool.cc

```diff
@@ -79,6 +80,21 @@ recordDiff(const Data* data, const FilterBase* filter, const ConfigSet& diff)
     }
 }
 
+void
+FilterPool::
+recordHitSegment(const Data* data, const FilterBase* f, FilterState & state){
+
+    FilterState::FilterReasons& reasons = state.getFilterReasons();
+    for ( auto it = reasons.begin() ; it != reasons.end(); ++it) {
+        for ( unsigned idx : it->second ){
+            const AgentConfig& config = *data->configs[idx].config;
+            events->recordHit("accounts.%s.filter.static.segment.%s",
```

Comment:
I'd replace the `segment` key fragment by the name of the filter. Makes the function suitable for other filters.